src/styles/main.scss

```diff
@@ -1,3 +1,262 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+$gray-elements: #f3f3f3;
+$yellow-stars: #ffde6a;
+$big_line-height: 18px;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.navbar {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  font-size: 12px;
+  text-transform: uppercase;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0,0,0,0.05);
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: $main-accent;
+
+    transition: 0.3s;
+
+    &:hover {
+      color: $blue-accent;
+    }
+  }
+
+  &__item {
+    display: block;
+    line-height: 60px;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.is-active {
+  color: $blue-accent;
+  position: relative;
+}
+
+.is-active:after {
+  content: "";
+
+  display: block;
+  position: absolute;
+  bottom: 0;
+
+  width: 100%;
+  height: 4px;
+
+  background-color: $blue-accent;
+  border-radius: 8px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border: 1px solid $gray-elements;
+  border-radius: 5px;
+
+  font-size: 10px;
+  line-height: 14px;
+  padding: 32px 16px 16px;
+
+  color: $main-accent;
+  background-color: $white;
+
+  margin: 0;
+
+  transition: transform 300ms ease-in-out;
+
+  &__image {
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__product-title {
+    margin: 0 0 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: $big_line-height;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+    color: $secondary;
+  }
+
+  &__review {
+    margin: 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    margin: 0;
+    color: $main-accent;
+    padding: 0;
+  }
```

Comment:
[Make use of SASS nesting (10-th point)](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md). Fix all cases please